hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/verifier/HandleContextVerifier.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.workflows.handle.verifier;
+
+import com.hedera.hapi.node.base.Key;
+import com.hedera.node.app.spi.signatures.SignatureVerification;
+import com.hedera.node.app.spi.workflows.VerificationAssistant;
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/**
+ * Helper class that contains all functionality for verifying signatures during handle.
+ */
+public interface HandleContextVerifier {
```

Comment:
This name doesn't seem just right. It isn't verifying a `HandleContext`. I'm not entirely clear on why this was factored out of `HandleContext` itself.

Reply:
Would `SignatureVerificationRespondent` work better?  This interface separates out the methods that respond to signature verification inquiries, as I understand it.